Example:  27%|█████████████████████▊                                                          | 831/3053 [00:22<00:50, 44.43it/s]Traceback (most recent call last):
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_frame.py", line 430, in _handle_exception
    self.do_wait_suspend(thread, frame, event, arg, exception_type=exception_type)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_frame.py", line 164, in do_wait_suspend
    self._args[0].do_wait_suspend(*args, **kwargs)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/pydevd.py", line 1937, in do_wait_suspend
    keep_suspended = self._do_wait_suspend(thread, frame, event, arg, suspend_type, from_this_thread, frames_tracker)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/pydevd.py", line 1972, in _do_wait_suspend
    time.sleep(0.01)
KeyboardInterrupt
Traceback (most recent call last):
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/site-packages/tqdm/std.py", line 1179, in __iter__
    yield obj
GeneratorExit
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_frame.py", line 930, in trace_dispatch
    self.do_wait_suspend(thread, frame, event, arg)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_frame.py", line 164, in do_wait_suspend
    self._args[0].do_wait_suspend(*args, **kwargs)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/pydevd.py", line 1937, in do_wait_suspend
    keep_suspended = self._do_wait_suspend(thread, frame, event, arg, suspend_type, from_this_thread, frames_tracker)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/pydevd.py", line 1972, in _do_wait_suspend
    time.sleep(0.01)
KeyboardInterrupt
Exception ignored in: <generator object tqdm.__iter__ at 0x7f89a3ea0050>
Traceback (most recent call last):
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/site-packages/tqdm/std.py", line 1192, in __iter__
    self.n = n
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/site-packages/tqdm/std.py", line 1192, in __iter__
    self.n = n
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_frame.py", line 930, in trace_dispatch
    self.do_wait_suspend(thread, frame, event, arg)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_frame.py", line 164, in do_wait_suspend
    self._args[0].do_wait_suspend(*args, **kwargs)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/pydevd.py", line 1937, in do_wait_suspend
    keep_suspended = self._do_wait_suspend(thread, frame, event, arg, suspend_type, from_this_thread, frames_tracker)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/_vendored/pydevd/pydevd.py", line 1972, in _do_wait_suspend
    time.sleep(0.01)
KeyboardInterrupt:
Traceback (most recent call last):
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/__main__.py", line 45, in <module>
    cli.main()
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/../debugpy/server/cli.py", line 444, in main
    run()
  File "/home/hwanhee/.vscode-server/extensions/ms-python.python-2021.5.842923320/pythonFiles/lib/python/debugpy/../debugpy/server/cli.py", line 285, in run_file
    runpy.run_path(target_as_str, run_name=compat.force_str("__main__"))
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "train.py", line 228, in <module>
    main()
  File "train.py", line 197, in main
    train_features = read(train_file, tokenizer, max_seq_length=args.max_seq_length)
  File "/home/hwanhee/project/correction/ATLOP/prepro.py", line 42, in read_docred
    tokens_wordpiece = tokenizer.tokenize(token)
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/site-packages/transformers/tokenization_utils.py", line 269, in tokenize
    (str(t), t) for t in self.all_special_tokens_extended if isinstance(t, AddedToken)
  File "/home/hwanhee/anaconda3/envs/atlop/lib/python3.7/site-packages/transformers/tokenization_utils_base.py", line 1080, in all_special_tokens_extended
    set_attr = self.special_tokens_map_extended
KeyboardInterrupt